clients/redigo.go

```diff
@@ -0,0 +1,387 @@
+package clients
+
+import (
+	"fmt"
+	"github.com/Shivam010/go-rejson/rjs"
```

Comment:
This will need to be changed : s/Shivam010/niitshm/ before pushing

Reply:
Yes-yes, I have that in my mind.
Each and every import paths will be rechecked and confirmed before the final push, don't worry about that.